docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -0,0 +1,252 @@
+```
+
+> **NOTE:** You might need to wait a few seconds for the Function to be ready.
+
+  </details>
+</div>
+
+## Create a Subscription with Event type consisting of alphanumeric characters
+
+Next, we will create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe for events of the type: `order.payment-success.v1`. Note that `order.payment-success.v1` contains a non-alphanumeric character (i.e. hyphen `-`).
+
+<div tabs name="Create a Subscription" group="create-subscription">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In Kyma Dashboard, go to the view of your Function `lastorder`.
+2. Go to **Configuration** > **Create Subscription+**.
+3. Provide the following parameters:
+   - **Subscription name**: `lastorder-payment-sub`
+   - **Application name**: `myapp`
+   - **Event name**: `order.payment-success`
+   - **Event version**: `v1`
+
+   - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.payment-success.v1`.
+
+4. Click **Create**.
+5. Wait a few seconds for the Subscription to have status `READY`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+```bash
+cat <<EOF | kubectl apply -f -
+   apiVersion: eventing.kyma-project.io/v1alpha1
+   kind: Subscription
+   metadata:
+     name: lastorder-payment-sub
+     namespace: default
+   spec:
+     sink: 'http://lastorder.default.svc.cluster.local'
+     filter:
+       filters:
+         - eventSource:
+             property: source
+             type: exact
+             value: ''
+           eventType:
+             property: type
+             type: exact
+             value: sap.kyma.custom.myapp.order.payment-success.v1
+EOF
+```
+
+To check that the Subscription was created and is ready, run:
+```bash
+kubectl get subscriptions lastorder-payment-sub -o=jsonpath="{.status.ready}"
+```
+
+The operation was successful if the returned status says `true`.
+  </details>
+</div>
+
+## Check the Subscription cleaned Event type
+
+To check the Subscription cleaned Event type, run:
+```bash
+kubectl get subscriptions lastorder-payment-sub -o=jsonpath="{.status.cleanEventTypes}"
+```
+
+Now, note that the returned cleaned event type `["sap.kyma.custom.myapp.order.paymentsuccess.v1"]` does not contain the hyphen (i.e. `-`) in `payment-success` part, because Eventing services cleans out the non-alphanumeric characters from the Event name and uses the cleaned Event name in underlying Eventing-Backend.
+
+## Trigger the workload with an event
+
+We created the `lastorder` Function, and subscribed to the `order.payment-success.v1` events by creating a Subscription CR. 
+Now we will see that we can still publish events with the original Event name (i.e. `order.payment-success.v1`) even though it contains the non-alphanumeric character, and it will trigger the Function.
```

Comment:
```suggestion
Next, you see that you can still publish events with the original Event name (i.e. `order.payment-success.v1`) even though it contains the non-alphanumeric character, and it will trigger the Function.
```